ci/scripts/build-chip-verilator.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Build a chip-level verilator simulation
+#
+# Expects three arguments: the toplevel to build, the fusesoc core and
+# the intermediate Verilated binary name
+
+set -e
+
+if [ $# != 3 ]; then
+    echo >&2 "Usage: build-chip-verilator.sh <toplevel> <fusesoc-core> <vname>"
+    exit 1
+fi
+tl="$1"
+fusesoc_core="$2"
+vname="$3"
+
+# Move to project root
+cd $(git rev-parse --show-toplevel)
+
+. util/build_consts.sh
+
+set -x
+
+mkdir -p "$OBJ_DIR/hw"
+mkdir -p "$BIN_DIR/hw/top_${tl}"
+
+# Compile the simulation without threading; the runners provided by
+# Azure provide two virtual CPUs, which seems to equal one physical
+# CPU (at most); the use of threading slows down the simulation.
```

Comment:
This comment is stale and doesn't match the "--threads 4" line below. (It's my fault, I forgot to delete the comment when switching the builder.)

Reply:
Duh! I didn't read that properly. Will ditch.